README.md

```diff
@@ -23,13 +23,21 @@ To install the component library itself
 yarn add suomifi-ui-components
 ```
 
-And include **required** styles and fonts from `dist/main.css` as best suited for your project. You can, for example, import the stylesheet to your app and let your bundler handle it:
+Include **required** styles from `dist/main.css` as best suited for your project. You can, for example, import the stylesheet to your app and let your bundler handle it:
 
 ```typescript
 import 'suomifi-ui-components/dist/main.css';
 ```
 
-This stylesheet contains the default fonts for the library, some global styles and Reach UI peer dependency styles.
+This stylesheet contains part of the global styles for the library and Reach UI peer dependency styles.
```

Comment:
If the sheet contains all of the Reach UI peer dependency styles so that the "part of" does not refer to them, I'd change this to "-- styles for the library **as well as** Reach UI peer dependency styles."

Reply:
Good point! Changed as suggested. The stylesheet contains all needed Reach-UI styles, but not all of them. So I guess that is all Reach UI styles from the perspective of this library => should be changed as proposed.